build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -74,7 +80,17 @@ object BuildParams {
     const val BUILD_VERSION_QUALIFIER = "versionQualifier"
     const val CI_ENVIRONMENT_VARIABLE = "CI"
     const val GRADLE_INSTALL_PATH = "gradle_installPath"
-    const val FLAKY_TEST_QUARANTINE = "flakyTestQuarantine"
+
+    /**
+     * Specify the flaky test quarantine strategy:
+     *
+     * -PflakyTest=include: run all tests, including flaky tests.
+     * -PflakyTest=exclude: run all tests, excluding flaky tests.
+     * -PflakyTest=quarantine: run flaky tests only.
+     *
+     * Default value (if absent) is "include".
+     */
+    const val FLAKY_TEST = "flakyTest"
```

Comment:
I'd like `runTests` better as a property. Then you'd have:
- `-PrunTests=all`
- `-PrunTests=noFlaky`
- `-PrunTests=onlyFlaky`

WDYT?

If you want to keep `flakyTest`, then I'd say something like this:
- `-PflakyTests=include`
- `-PflakyTests=exclude`
- `-PflakyTests=only`

Reply:
I would like it to reflect "it's a strategy about flaky controlling", so I use `-PflakyTests=include/exclude/only` now.